resources/js/views/settings/Application.vue

```diff
@@ -166,139 +166,212 @@
+              :state='fieldState("pagination_page_size")'
+              :label="$t('settings.application.paginationPageSize.title')"
+            >
+              <b-form-input id="application-pagination-page-size-input"
+                            v-model="settings.pagination_page_size"
+                            type="number"
+                            :disabled="isBusy || viewOnly || !loaded"
+                            min="1"
+                            max="100"
+                            required
+                            :state='fieldState("pagination_page_size")'
+              >
+              </b-form-input>
 
-          <template slot='invalid-feedback'>
-            <div v-html="fieldError('password_self_reset_enabled')"></div>
-          </template>
-        </b-form-group>
+              <template slot='invalid-feedback'>
+                <div v-html="fieldError('pagination_page_size')"></div>
+              </template>
+            </b-form-group>
+          </b-col>
+
+          <b-col md='6'>
+            <!--Own rooms pagination page size settings-->
+            <b-form-group
+              label-class="font-weight-bold"
+              class="mb-4"
+              label-for="application-pagination-own-room-page-size-input"
+              :description="$t('settings.application.ownRoomsPaginationPageSize.description')"
+              :state='fieldState("own_rooms_pagination_page_size")'
+              :label="$t('settings.application.ownRoomsPaginationPageSize.title')"
+            >
+              <b-form-input id="application-pagination-own-room-page-size-input"
+                            v-model="settings.own_rooms_pagination_page_size"
+                            type="number"
+                            min="1"
+                            max="25"
+                            required
+                            :disabled="isBusy || viewOnly || !loaded"
+                            :state='fieldState("own_rooms_pagination_page_size")'
+              >
+              </b-form-input>
 
-        <b-form-group
-          label-class="font-weight-bold"
-          class="mb-4"
-          :label="$t('settings.application.default_timezone')"
-          label-for='timezone'
-          :state='fieldState("default_timezone")'
-        >
-          <b-input-group>
-            <b-form-select
-              :options='timezones'
-              id='timezone'
-              v-model='settings.default_timezone'
+              <template slot='invalid-feedback'>
+                <div v-html="fieldError('own_rooms_pagination_page_size')"></div>
+              </template>
+            </b-form-group>
+          </b-col>
+        </b-row>
+
+        <b-row cols='12'>
+          <b-col md='6'>
+            <b-form-group
+              label-class="font-weight-bold"
+              class="mb-4"
+              :label="$t('settings.application.default_timezone')"
```

Comment:
wrong naming convention. We don't have it equal through the whole app, but let's try to at least use the same standard in a component/view.